lib/experimental/fonts-api/deprecations/class-wp-web-fonts.php

```diff
@@ -690,7 +690,7 @@ public function to_theme_json( $font_family_handle ) {
 			}
 
 			$variation_obj        = $this->registered[ $variation_handle ];
-			$variation_properties = array( 'origin' => 'gutenberg_wp_webfonts_api' );
+			$variation_properties = array( 'origin' => 'gutenberg_wp_fonts_api' );
```

Comment:
Is there risk of this impacting extenders who may be iterating through the registered fonts? 

A search of the wporg plugin directory shows only Gutenberg using it https://wpdirectory.net/search/01GSZ95M6F3772YGWKZXG9A15P.

I suspect the risk to users is minimally low.

Reply:
If anything, this feature is in the `experimental` folder so external developers should be aware that things might break 😅 

Is it worth it to include a `doing_it_wrong` call to announce that things changed, @hellofromtonya?

Reply:
>If anything, this feature is in the experimental folder so external developers should be aware that things might break 😅
True, though the API now includes a temporary BC Layer which is intended to keep sites running while giving time for extenders to upgrade to the new API.

>Is it worth it to include a doing_it_wrong call to announce that things changed, @hellofromtonya?

Good question @zaguiini! The origin is set internally within the API itself. So I think it's okay to not add a `doing_it_wrong` here. But it does me think about: could it be set externally? If yes and set wrong, what happens? Hmm that's interesting as it may cause unexpected things. Thinking a new issue is needed to discuss and explore.